assets/js/modules/analytics/datastore/__fixtures__/index.js

```diff
@@ -42,12 +42,6 @@ export { default as dashboardBounceRateWidgetData } from './dashboard-bounce-rat
 export { default as dashboardBounceRateWidgetArgs } from './dashboard-bounce-rate-widget-args.json';
 export { default as dashboardGoalsWidgetData } from './dashboard-goals-widget-data.json';
 export { default as dashboardGoalsWidgetArgs } from './dashboard-goals-widget-args.json';
-export { default as dashboardUserDimensionsData } from './dashboard-user-dimensions-data.json';
-export { default as dashboardUserDimensionsArgs } from './dashboard-user-dimensions-args.json';
-export { default as dashboardUserTotalsData } from './dashboard-user-totals-data.json';
-export { default as dashboardUserTotalsArgs } from './dashboard-user-totals-args.json';
-export { default as dashboardUserGraphData } from './dashboard-user-graph-data.json';
-export { default as dashboardUserGraphArgs } from './dashboard-user-graph-args.json';
 export { default as wpDashboardPopularPagesArgs } from './wp-dashboard-popular-pages-args.json';
```

Comment:
If it's trivial, it would be great if you could also migrate the WP dashboard stories to using this new function, since those fixtures were added in the meantime.

Reply:
Yep, updated.